docs/visualization/plotting-1d-data.ipynb

```diff
@@ -413,7 +413,7 @@
     "d4['Sample'].masks['mask1'] = sc.Variable(dims=['tof'],\n",
     "                                          values=np.where(np.abs(x-40) < 10, True, False))\n",
     "d4['Background'].masks['mask1'] = ~d4['Sample'].masks['mask1']\n",
-    "d4['Background'].masks['mask1']['tof', 0:20].values = np.zeros(20, dtype=np.bool)\n",
+    "d4['Background'].masks['mask1']['tof', 0:20].values = np.zeros(20, dtype=bool)\n",
```

Comment:
Also, numpy is now deprecating the use of it's internal types `np.bool` -> `bool`.  FYI https://numpy.org/devdocs/release/1.20.0-notes.html

Reply:
yes, that is the reason for this change

Reply:
😆 I need to read more carefully